lib/routes/gwyzzjg/articles.ts

```diff
@@ -0,0 +1,71 @@
+import { Route } from '@/types';
```

Comment:
Please place the route under `gov` folder

Reply:
Thanks for your review, the revision has been done.